package/android/src/newarch/java/com/reactnativecommunity/slider/ReactSliderManager.java

```diff
@@ -40,10 +40,17 @@ protected ViewManagerDelegate<ReactSlider> getDelegate() {
           new SeekBar.OnSeekBarChangeListener() {
             @Override
             public void onProgressChanged(SeekBar seekbar, int progress, boolean fromUser) {
+              ReactSlider slider = (ReactSlider)seekbar;
+
+              if(slider.getLimit() > 0 && progress > slider.getLimit()) {
```

Comment:
What if `minimumValue = -100`, `maximumValue = -50` and `limit = -70`?
I'm afraid we have to consider the limit to be both positive and negative values including 0.
So to know whether Slider is limited we should introduce the internal field of Slider object: `boolean isLimited` which would be `false` as default and set to `true` if any value is passed to `limit` prop.

Reply:
Oops I forgot about negative values 😅. That makes sense. 
There's a problem with the `isLimited` approach. Since number is `0` by default when it gets to native, I couldn't find a way to identify if a value from JS is `undefined` or actually `0`. I'm changing my approach to set default limits to very small (for `lowerLimit`) and very big (for `upperLimit`) numbers. That way I don't need a boolean flag to limit progress.